assets/js/blocks/cart-checkout/cart/full-cart/utils.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+import { PLACEHOLDER_IMG_SRC } from '@woocommerce/block-settings';
+
+/**
+ * Return a currency value as a number for doing calculations.
+ * Note this doesn't convert into dollars, currency values are in minor units (e.g. cents).
+ *
+ * @param {string} currencyValue Currency value string (in minor unit).
+ * @return {number} The currency value as int (in minor unit).
+ */
+export const getPriceNumber = ( currencyValue ) => {
+	return parseInt( currencyValue, 10 );
+};
+
+/**
+ * Returns a formatted element containing variation details.
+ */
+export const ProductVariationDetails = ( { variation } ) => {
```

Comment:
:100: for moving these components out of `cart-line-item-row.js`, but personal opinion here: I expect `utils.js` to be for util functions, while I would expect each component to be in its own file. I know that means we will end up with many small files, but I usually prefer that than having to search inside files, it also makes reusing the components easier.

But I don't have a strong opinion, so I'm fine either way. :+1: